controllers/akodeploymentconfig/phases/phases.go

```diff
@@ -92,6 +91,9 @@ func ReconcileClustersPhases(
 				cluster.GroupVersionKind(), cluster.Namespace+"/"+cluster.Name)
 		}
 
+		// update cluster avi label before run any phase functions
+		ako_operator.ApplyClusterLabel(log, &cluster, obj)
```

Comment:
Should this move to line 92 inside loop?

Reply:
It is already inside the loop, right? Line 92 inits patcher for updating cluster object in db.

Reply:
Ah, sorry. The indentation was off in the web UI which confused me